content/legal/release-policy.md

```diff
@@ -44,7 +44,37 @@ requirements of ASF policy on releases as described below, validate all
 cryptographic signatures, compile as provided, and test the result on their
 own platform.
 
-Release votes SHOULD remain open for at least 72 hours.
+Release votes SHOULD remain open for at least 72 hours. See
+[RFC 2119](https://www.rfc-editor.org/rfc/rfc2119) for a good definition of
+SHOULD and the next [Expedited Releases](#expedited-releases) section when
+considering a reduced voting period.
+
+#### Expedited Releases {#expedited-releases}
+As stated above, the normal policy for releases is to allow 72 hours for
+release reviews and votes, however the review/voting period for a release
+can be reduced in exceptional circumstances.
+
+ASF projects are made up of distributed teams, in multiple time zones and volunteers
+with lives and jobs and the rationale behind 72 Hours is to try and give all
+members of a project the opportunity to take part in the decision to release.
+
+The most obvious example of an exceptional circustamce would be for a fix for a
+publicly known security issue. Everyone will probably have a different definition
+of what an exceptional circumstance is, but ultimately it is down to individual
+PMCs to decide for their project. 
+
+Projects SHOULD give as much notice as possible for an expedited release in
+order to give project members a chance to make time to participate in the
+decision.
```

Comment:
I think it would be worth mentioning that any committer might veto expedited release if they feel there is not enought time time to vote.

Reply:
I don't think we should introduce veto here. It creates more special circumstances for releasing.

The voting process says release vote follows majority approval. And as you said here, the RM would have a final say.

Reply:
Yep. Agreed.

Reply:
A) Committers can't veto anything as only PMC members have power and B) releases can't be vetoed